validation/standard/standard-version-consistency_test.go

```diff
@@ -0,0 +1,56 @@
+package standard_test
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation"
+	"github.com/ethereum-optimism/superchain-registry/validation/standard"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/require"
+)
+
+// This test will check:
+// For each each superchain
+// For each contract release
+// For each contract version declaration which specifies an address
+// There is a contract at that address with the matching bytecode and semver.
+// This is a consistency check on the standard package itself, not any particular chain
+func TestStandardVersionConsistency(t *testing.T) {
+	for _, superchain := range []string{"sepolia", "mainnet"} {
```

Comment:
Can we make this less brittle so that if/when a new superchain is added it automatically works?

Reply:
Can do this in a follow-up. We have the same brittleness in a few other spots